deploy-board/deploy_board/templates/configs/new_capacity.html

```diff
@@ -171,7 +171,9 @@ <h4 class="panel-title pull-left pointer-cursor">Basic Settings</h4>
                         window.location.href='/env/'+environment.envName+'/'+environment.stageName+'/config/capacity/'
                     },
                     error: function (data) {
-                        globalNotificationBanner.error = data
+                        console.log("new_cap");
```

Comment:
`console.error()`?

Reply:
Thanks for the code review. I removed the output in console. ^^